### PR TITLE
[HttpKernel][FrameworkBundle] Fix escaping of serialized payloads passed to test clients

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 2.8
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #28125
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Review by fabpot — APPROVED** [review]

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Bundle/FrameworkBundle/Client.php [hunk 1]
```diff
@@ -187,16 +187,16 @@ protected function getScript($request)
 
 error_reporting($errorReporting);
 
-if ('$autoloader') {
-    require_once '$autoloader';
+if ($autoloader) {
+    require_once autoloader;
```
[the remark this answers is not in the document]
Comment (nicolas-grekas) [reply src/Symfony/Bundle/FrameworkBundle/Client.php]: oups, fixed thx

**Comment by fabpot** [comment]
Looks like tests are broken with this change.

**Comment by nicolas-grekas** [comment]
Fixed, should be green soon.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**